fortanix-vme/nitro-attestation-verify/src/mbedtls.rs

```diff
@@ -0,0 +1,109 @@
+use aws_nitro_enclaves_cose::crypto::{Hash, MessageDigest, SignatureAlgorithm, SigningPublicKey};
```

Comment:
do we need to add copyright to these?

Reply:
I've added a LICENSE file, but I have zero experience with this. Is that sufficient to meet the expectations of [their Apache 2.0 license](https://github.com/awslabs/aws-nitro-enclaves-cose/blob/main/LICENSE)? Is it even necessary?

Reply:
```
/* Copyright (c) Fortanix, Inc.                                                                                                                                                                                                          
 *                                                                                                                                                                                                                                       
 * This Source Code Form is subject to the terms of the Mozilla Public                                                                                                                                                                   
 * License, v. 2.0. If a copy of the MPL was not distributed with this                                                                                                                                                                   
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Putting this at the top of each file should be sufficient.

Reply:
You mean as well? @jethrogb do you have experience with this? Do we need to do anything special since we use the `aws_nitro_enclaves_cose` crate?

Reply:
Added the copyright statements to `lib.rs` and `mbedtls.rs`